linera-chain/src/lib.rs

```diff
@@ -71,15 +71,24 @@ pub enum ChainError {
         next_height: BlockHeight,
         next_index: u32,
     },
-    #[error("Block proposed to {chain_id:?} is attempting to skip unskippable message {event:?}")]
-    UnskippableMessage {
+    #[error("Block proposed to {chain_id:?} is attempting to reject protected message {event:?}")]
+    CannotRejectMessage {
         chain_id: ChainId,
         origin: Box<Origin>,
         event: Event,
     },
     #[error(
-        "Incoming message in block proposed to {chain_id:?} has timestamp {message_timestamp:},
-        which is later than the block timestamp {block_timestamp:}."
+        "Block proposed to {chain_id:?} is attempting to skip a message \
+             that cannot be skipped: {event:?}"
```

Comment:
```suggestion
         that cannot be skipped: {event:?}"
```

Reply:
I believe `rustfmt` did that. Let me try again.